fastparquet/converted_types.py

```diff
@@ -115,6 +115,7 @@ def convert(data, se):
     elif ctype == parquet_thrift.ConvertedType.INTERVAL:
         # for those that understand, output is month, day, ms
         # maybe should convert to timedelta
+        # TODO: seems like a np.view should do this much faster
```

Comment:
I was curious what was going on here so I inserted a PDB statement and ran tests to trigger a situation where this code would be active with data.  It looks like this code wasn't covered by the test suite.